### ISSUE TITLE
Cannot set AWS_CACHE_BUCKET_READ environment variable

### ISSUE THREAD

**Issue by dein0s** [aws-amplify/amplify-hosting#1933]
**Please describe which feature you have a question about?**
Due to some cache bloat issues we were searching on how to temporary disable it without overriding cache paths and found `AWS_CACHE_BUCKET_READ` env variable in FAQ
https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md#how-do-i-disable-reading-from-cache

Unfortunately, it seems to be impossible to set this variable because of the reserved "AWS" prefix.

**Provide additional details**

**What AWS Services are you utilizing?**
AWS Amplify

**Provide additional details e.g. code snippets**
![image](https://user-images.githubusercontent.com/7908313/121145463-a61bb900-c847-11eb-921c-169343790c07.png)

**Comment by dein0s** [comment]
App ID: `d67fy3h9m237g`
Region: `us-east-1`

**Comment by houmark** [comment]
Can confirm the same happened for me. I felt like it saved the first time (green confirmation bar maybe?!). But the variable didn't show up. Refreshed hard. I then retried saving and it threw that error. I also removed the cache from the build pipeline (our cache on an old platform was more than 5TB!!).

Building then threw this error:
`2021-06-14T05:10:59.442Z [WARNING]: !TypeError: Cannot read property 'paths' of undefined`

It does look like it finished building though (and it was much faster as it didn't have to battle with a huge cache that seems to just have increased over time and probably also had the node_modules cache).
